### ISSUE TITLE
Align RNASeq samples to genome using STAR for diff. expression analysis

### ISSUE THREAD

**Issue by laurahspencer** [laurahspencer/DuMOAR#32]
When you're done @ggoetznoaa can you drop your script and the resulting gene count matrix into this repo? Also please provide/confirm which annotation file you used. Thanks!

**Comment by ggoetznoaa** [comment]
@laurahspencer Ok, I've finished running STAR again against the reference using the GENSAS annotation. I kept the gene counts and I used the default settings. Is there are particular folder you want me to put the gene count matrix file and script I used for aligning?

Also to answer your question, here is what I used for the STAR alignment
reference fasta file
PGA_assembly.scaffolds_only.fasta

annnotation file
Metacarcinus-magister-v1.0.a1.6448195bde6ca-publish.genes.scaffolds_only.gtf
